mne/viz/_brain/surface.py

```diff
@@ -122,6 +123,18 @@ def load_geometry(self):
                 path.join(self.data_path, 'surf',
                           '%s.%s' % (self.hemi, self.surf)))
             orig_faces = faces
+        self._make_geometry(coords, faces, orig_faces)
+        # load in curvature values from the ?h.curv file.
+        curv_path = path.join(self.data_path, 'surf', '%s.curv' % self.hemi)
+        curv = read_curvature(curv_path, binary=False)
+        self._make_curvature(curv)
```

Comment:
I wouldn't add anything new to `_Surface` -- this is really for brain left/right hemi surfaces (feel free to update the docs to make this clear). For any other surfaces (including the marching-cube-created surfaces) they can be plotted directly using the renderer mesh methods.